packages/gcp/src/lib/ServiceWrapper.ts

```diff
@@ -110,13 +110,15 @@ export default class ServiceWrapper {
     }
   }
 
-  private extractZones(items: InstanceData): string[] {
-    if (!items) return []
-    return Object.entries(items)
-      .filter((zone: Zone) => {
-        return zone[1].warning?.code !== this.noResultsOnPageMessage
-      })
-      .map((zone) => zone[0].replace('zones/', '').replace('regions/', ''))
+  private async extractZones(results: IterableScopedList): Promise<string[]> {
```

Comment:
this update was to work with the iterable changes?

Reply:
Yes, the `aggregatedListsAsync` functions used on lines 81-87 return async iterables. Therefore, the arguments of this function where those results are used needed to be updated to match that. (Including the logic for looping over the results)